src/.env

```diff
@@ -0,0 +1,4 @@
+OPENAI_API_VERSION=
+AZURE_OPENAI_ENDPOINT=
+AZURE_OPENAI_DEPLOYMENT=
+AZURE_OPENAI_API_KEY=
```

Comment:
Why does the sample code use keys? Shouldn't we be moving to managed identity for our starter templates?

Reply:
Updated.